app/lib/dags/non_host_alignment.json.erb

```diff
@@ -22,12 +22,26 @@
       "gsnap.hitsummary.tab",
       "gsnap_counts.json"
     ],
+    "refined_gsnap_out": [
+      "gsnap.blast.m8",
+      "gsnap.reassigned.m8",
+      "gsnap_refined.hitsummary.tab",
+      "refined_gsnap_counts.json",
```

Comment:
maybe gsnap_refined_out and gsnap_refined_counts.json / rapsearch etc. for parallelism?